### PR TITLE
Implement toolbar dividers

### PR DESCRIPTION
**Pull request by BoardJames**
This pull request adds toolbar dividers:
toolbar-divider

They were added to match the mockup here: https://github.com/WordPress/gutenberg/issues/972#issuecomment-305689289

### CODE REVIEW

**Comment by jasmussen** [comment]
NICE! 

I took the liberty of pushing a teensy bit of polish to the divider itself. Code also looks good to me —  @youknowriad any concerns? Otherwise 👍

**Review by youknowriad — APPROVED** [review]
Nice 🚢

File: blocks/library/freeform/freeform-block.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ const FREEFORM_CONTROLS = [
 		title: wp.i18n.__( 'Convert to ordered' ),
 	},
 	{
+		showDivider: true,
```
Comment (aduth) [inline blocks/library/freeform/freeform-block.js]: At a glance, it's not entirely obvious to me where the divider would be shown. To the left or right?

Comment (EphoxJames) [inline blocks/library/freeform/freeform-block.js]: Good point. It shows to the left so I suppose the simple solution is to rename it to leftDivider.

**Comment by EphoxJames** [comment]
I think I've addressed all concerns so I'll merge this.

**Pull request merged**